hs_core/templates/pages/my-resources.html

```diff
@@ -56,15 +56,19 @@ <h2 class="page-title">Resources</h2>
             </table>
 
             <div class="btn-toolbar">
-                <a class="btn btn-link" href="?start=0"><i class="fa fa-fast-backward"></i></a>
-                <a class="btn btn-link" href="?start={{ first|add:"-20" }}"><i class="fa fa-backward"></i></a>
+                <a {% if first == 0 %} disabled {% endif %}
+                        class="btn btn-link" href="?start=0"><i class="fa fa-fast-backward"></i></a>
+                <a {% if first == 0 %} disabled {% endif %}
+                        class="btn btn-link" href="?start={{ first|add:"-20" }}"><i class="fa fa-backward"></i></a>
                 {% if last %}
                     Results {{ first|add:"1" }}-{{ last }} of {{ ct }}
                 {% else %}
                     Results 0-0 of 0
                 {% endif %}
-                <a class="btn btn-link" href="?start={{ first|add:"20" }}"><i class="fa fa-forward"></i></a>
-                <a class="btn btn-link" href="?start={{ ct|add:"-20" }}"><i class="fa fa-fast-forward"></i></a>
+                <a {% if last == ct %} disabled {% endif %}
```

Comment:
I know, we are on the same page regarding valid values for first and last. I was just thinking to have the check a bit more relaxed to make sure to capture all invalid cases, although they may not occur since our validation code ensures this. It will cover the case you want to check any way as well as all potentially invalid cases, so thought making it a bit more relaxed won't hurt, although it is not needed if our validation code makes sure invalid cases will not occur.